crates/cosmos/cosmos-chain-components/src/types/config/gas/dynamic_gas_config.rs

```diff
@@ -0,0 +1,43 @@
+use crate::types::config::gas::eip_type::EipQueryType;
+
+#[derive(Debug, Clone, PartialEq, PartialOrd)]
+pub struct DynamicGasConfig {
+    pub enabled: bool,
```

Comment:
I think we don't need the `enabled` flag here, and use `Option` inside `GasConfig` to indicate that it is disabled.

Reply:
Done in 05bb04f4939bd3957461c2b48ae93484bb89a697